.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -999,6 +934,85 @@ protected double GetPointValue(List<string> matchStrs)
             return ret;
         }
 
+        protected int GetSplitIndex(List<string> fracWords)
+        {
+            var splitIndex = fracWords.Count - 1;
+            var currentValue = Config.ResolveCompositeNumber(fracWords[splitIndex]);
+            long roundValue = 1;
+            for (splitIndex = fracWords.Count - 2; splitIndex >= 0; splitIndex--)
+            {
+                if (Config.WrittenFractionSeparatorTexts.Contains(fracWords[splitIndex]) ||
+                    Config.WrittenIntegerSeparatorTexts.Contains(fracWords[splitIndex]))
+                {
+                    continue;
+                }
+
+                var previousValue = currentValue;
+                currentValue = Config.ResolveCompositeNumber(fracWords[splitIndex]);
+
+                var hundredsSM = 100;
+
+                // Below flag isUncomposobleWithSeparator is used to handle one existing bug for handling fraction input like "two and fifty-four hundredths".

```

Comment:
Can you change explanation to what it currently is rather than ask to compare with previous code ? Explanation should be about code and not ask the person reading code to compare with previous code according to me.

Reply:
Updated comments, please help to check again.